db/migrations/0003_alter_movie_actors_alter_movie_genres_and_more.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 5.1.4 on 2024-12-30 15:37
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_cinemahall_movie_moviesession'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='actors',
+            field=models.ManyToManyField(related_name='actors', to='db.actor'),
+        ),
+        migrations.AlterField(
+            model_name='movie',
+            name='genres',
+            field=models.ManyToManyField(related_name='genres', to='db.genre'),
+        ),
+        migrations.AlterField(
+            model_name='moviesession',
+            name='cinema_hall',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='halls', to='db.cinemahall'),
+        ),
+        migrations.AlterField(
+            model_name='moviesession',
+            name='movie',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movies', to='db.movie'),
```

Comment:
The `related_name` for the `movie` field is set to 'movies', which might not clearly indicate the relationship. Consider using a more descriptive related name to better reflect the relationship.